src/pyfaf/actions/repoassign.py

```diff
@@ -41,8 +41,15 @@ def run(self, cmdline, db):
         opsys_list = []
 
         for item_name in cmdline.OPSYS + cmdline.ARCH:
+            
             opsys = get_opsys_by_name(db, item_name)
             arch = get_arch_by_name(db, item_name)
+            
+            #If name is rhel we will search Red Hat Enterprise Linux
+            if not (opsys or arch):
+                if item_name == "rhel":
+                    item_name = "Red Hat Enterprise Linux"
+                    opsys = get_opsys_by_name(db, item_name)
 
```

Comment:
This doesn't seem right at all. Fixing rhel -> Red Hat Enterprise Linux ambiguity in one place (case) only?